PSKoans/Koans/Foundations/AboutStrings.Koans.ps1

```diff
@@ -47,8 +47,7 @@ Describe 'Strings' {
         }
 
         It 'can do a simple expansion' {
-            $Windows = Get-Item 'C:\Windows' | Select-Object -ExpandProperty FullName
-            '__' | Should -Be "The windows directory is located here: $Windows"
+            '__' | Should -Be "Your working directory is located here: $pwd"
```

Comment:
$pwd isn't a bad choice, but it is one that will cause problems if the end user tries to run `Measure-Karma` from another directory. Additionally, there isn't actually a guarantee that the koans will be evaluated with the same current working directory; the koans are evaluated in subprocesses that are completely removed from the user's own session.

Using a more consistent variable such as $home may be more preferable, or $env:TEMP